frontend/plan/src/components/schedule/Schedule.js

```diff
@@ -228,6 +235,7 @@ const mapStateToProps = state => (
 const mapDispatchToProps = dispatch => (
     {
         removeSection: idDashed => dispatch(removeSchedItem(idDashed)),
+        focusSection: id => dispatch(fetchSectionInfo({ param: id })),
```

Comment:
I understand that requirements and associated section have yet to be implemented in the backend, but just wanted to point out that we might want to think about how to separate the information that users get from clicking on a course versus the information that users get from clicking on a section. Right now the same information is displayed when clicking on a course from `search_results.js` and any one of the blocks, while nothing is shown when clicking on a section in the `SectionList` component.

Reply:
That's a really good point!